README.md

```diff
@@ -25,3 +25,29 @@ npm run build
 ```
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+## Internationalization (i18n)
+
+You are welcome to submit translations of the wiki pages. So far we only have a small portion of the wiki translated to French. We currently only have capacity to review the following languages:
+
+- French
+- German
+- Swedish
+- Turkish
+
+If your translation is a mere Google translation without any correction, the Pull Request will likely not be accepted.
+
+**Before you begin translating, please consider your submission as *commitment* to maintain the pages in sync with any changes happening in the english version**
+
+### Enable a new language for translation
+
+1. Create a folder for the documents inside the `i18n` folder. For example german: `mkdir -p i18n/<two-letter-international-code>/docusaurus-plugin-content-docs/current` (*note that you will need replace `<two-letter-international-code>` with the actual two-letter international code*).
```

Comment:
Where can the translator find the list of `<two-letter-international-code>`?

Reply:
Good catch, thought i had included it:
https://www.loc.gov/standards/iso639-2/php/English_list.php

Should probably replace `two-letter-international-code>` with `<ISO639>` also

Reply:
added